kin-sdk/kin-sdk-lib/src/main/java/kin/sdk/KinAccount.java

```diff
@@ -81,7 +81,7 @@
      * @throws AccountNotFoundException if the sender or destination account was not created.
      * @throws OperationFailedException other error occurred.
      */
-    Transaction buildTransactionSync(@NonNull String publicAddress, @NonNull BigDecimal amount, int fee) throws OperationFailedException;
+    PaymentTransaction buildTransactionSync(@NonNull String publicAddress, @NonNull BigDecimal amount, int fee) throws OperationFailedException;
```

Comment:
maybe need to change method name as well, to `buildPayment` or `buildSendKin` like in server? or it's too much

Reply:
If we already change then no problem will change it also